jakartaee9/src/web-facelettaglibrary_3_0.xsds

```diff
@@ -35,17 +35,17 @@
   <xsd:annotation>
     <xsd:documentation>
       <![CDATA[[
-      <p>The XML Schema for the Tag Libraries in the JavaServer Faces
+      <p>The XML Schema for the Tag Libraries in the Jakarta Server Faces
       Standard Facelets View Declaration Language (Facelets VDL)
-      (Version 2.3).</p>
+      (Version 3.0).</p>
       
-      <p>JSF 2.3 Facelet Tag Libraries that wish to conform to this
-      schema must declare it in the following manner.</p>
+      <p>Jakarta Server faces 3.0 Facelet Tag Libraries that wish to conform to 
```

Comment:
```suggestion
      <p>Jakarta Server Faces 3.0 Facelet Tag Libraries that wish to conform to 
```
In this file references to JSF could be shortened to just Faces instead of Jakarta Server Faces. e.g., Faces Behavior, Faces UIComponent, Faces Converter, Faces Validator. 
I will let others comment on this. (@arjantijms)